activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -467,7 +471,12 @@ def new_column_definition(name, type, **options) # :nodoc:
       end
 
       private
+        def valid_column_definition_option_keys
```

Comment:
If we need to backport any keys for compatibility, we can override this method in https://github.com/rails/rails/blob/master/activerecord/lib/active_record/migration/compatibility.rb